packages/components/src/calendar-react-day-picker/styles.scss

```
@@ -0,0 +1,448 @@
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+@import "../styles/typography";
+
+/* Root of the component. */
+.a8c-components-calendar {
+  // TODO:
+  // - move`font-family` rule (and "../styles/typography" import) out of component
+  // - add `font-family` rule to Storybook's global styles
+  // - then 	Theme is ready and includes type styles, add back to component
+  font-family: $a8c-font-family-sans;
+
+  // Internal variables
+  --a8c-components-calendar-outline-focus: 1.5px solid var(--color-accent);
+  --a8c-components-calendar-button-height: 2rem;
+  --a8c-components-calendar-button-width: 2rem;
+  --a8c-components-calendar-nav-height: 2rem;
+
+  // TODO: themify / align to DS tokens
+  --a8c-components-calendar-range-middle-background-color: color-mix(in srgb, var(--color-accent) 4%, transparent);
+  --a8c-components-calendar-preview-border-color: color-mix(in srgb, var(--color-accent) 16%, transparent);
+
+
+  position: relative; /* Required to position the navigation toolbar. */
+  box-sizing: border-box;
+  display: inline flow-root;
+  color: $gray-900;
+  font-size: 13px;
+  font-weight: 400;
+
+  *, *::before, *::after {
+    box-sizing: border-box;
+    color: inherit;
+    font: inherit;
+  }
+
+  &__day {
```

Personally I'm a proponent of spelling out the full class name to make it easier to search for in code.

```suggestion
}

.a8c-components-calendar__day {
```

---

I expanded the selectors. It gets a bit verbose, but I do agree what code search is a lot better this way